appeals/api/src/server/repositories/service-user.repository.js

```diff
@@ -0,0 +1,21 @@
+import { databaseConnector } from '#utils/database-connector.js';
+
+/** @typedef {import('@pins/appeals.api').Schema.ServiceUser} ServiceUser */
+/**
+ * @typedef {import('#db-client').Prisma.PrismaPromise<T>} PrismaPromise
+ * @template T
+ */
+
+/**
+ * Updates a service user's details
+ * @param { number } id
+ * @param {{organisationName: string | null, firstName: string, middleName: string | null, lastName: string, email: string | null, phoneNumber: string | null, addressId: number | null }} data
```

Comment:
You can avoid re-writing these types and use the Prisma type, this will ensure its compatible with the update call too, e.g. `import('#db-client').Prisma.ServiceUserUpdateInput`